test/scripts/openshift-ci/infra/deploy.gateway.ingress.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
We do not have license headers in other shell scripts. Do we need it? Isn't it inherited from `LICENSE` in the root of the project?

Reply:
I see your point, but I think the logic is a bit inconsistent. We add license headers to .go and .py files to make the license clear when individual files are copied or shared. Shell scripts are still code, so the same reasoning could apply. It’s just that in practice, short utility scripts often don’t have headers and rely on the root LICENSE file, which might be why they’re treated differently. That said, if you prefer, it’s fine to leave out the license headers in the scripts.